### ISSUE TITLE
Invoke-CMApplyDriversPackage.ps1 v4 - Unable to determine AdminService endpoint type, bailing out

### ISSUE THREAD

**Issue by m0rph0** [MSEndpointMgr/ConfigMgr#261]
Environment: MECM 2002, system account created, variables created, v4.0 script.

Running script in DebugMode (using service account credentials), doesn't seem to do anything.. 
Any idea how to troubleshoot?

**Comment by Ar5had07** [comment]
Yeah same thing here, I'm not sure what the endpoint is.  If i put the primary site server in the endpoint, i get the error

 - Failed to retrieve available package items from AdminService endpoint. Error message: The remote server returned an error: (401) Unauthorized.	ApplyDriverPackage	14/08/2020 14:07:48	13676 (0x356C)
 - An error occurred while calling AdminService for a list of available driver packages. Error message: Cannot validate argument on parameter 'Message'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.	ApplyDriverPackage	14/08/2020 14:07:48	13676 (0x356C)
[ApplyDriverPackage]: Apply Driver Package process failed, please refer to previous error or warning messages	ApplyDriverPackage	14/08/2020 14:07:48	13676 (0x356C)

I've tried the CMG FQDN, still the same error.

What is supposed to go in the Endpoint switch?

**Comment by NickolajA** [comment]
It's supposed to be the FQDN of the server where the SMS Provider is running, as the AdminService is installed on that. It's also written in our updated docs:

See step 4:

https://msendpointmgr.com/modern-driver-management/
